src/psyclone/dynamo0p3.py

```diff
@@ -7212,13 +7212,13 @@ def gen_code(self, parent):
 
         '''
         # Check iteration space before generating code
-        if self._iterates_over not in USER_KERNEL_ITERATION_SPACES:
+        if self.iterates_over not in USER_KERNEL_ITERATION_SPACES:
             raise GenerationError(
                 "The LFRic API supports calls to user-supplied kernels "
                 "that have one of {0} as iteration space, but kernel '{1}' "
                 "has an iteration space of '{2}'.".
                 format(USER_KERNEL_ITERATION_SPACES, self._name,
-                       self._iterates_over))
+                       self.iterates_over))
```

Comment:
and the `self._name` on the line above, pretty please :-)

Reply:
Done.